packages/@ourworldindata/grapher/src/stackedCharts/StackedUtils.ts

```diff
@@ -18,19 +26,36 @@ export const stackSeries = <PositionType extends StackedPointPositionType>(
     return seriesArr
 }
 
+// Makes sure that values are evenly spaced
+function withUniformSpacing(values: number[]): number[] {
+    const deltas = values
+        .slice(0, -1)
+        .map((xVal, index) => values[index + 1] - xVal)
+    const gcd = findGreatestCommonDivisorOfArray(deltas)
```

Comment:
Need to include a check that `deltas.length >= 1` here (or in `findGreatestCommonDivisorOfArray` itself), otherwise this goes haywire for single-year charts.